diagnostic.md

```diff
@@ -99,7 +117,7 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+text color will be yellow, the font-family will be sans-serif.
 <!-- your answer ends here -->
```

Comment:
font family will actually be cursive because the inline specificity of that styling is always the most powerful